core/src/main/java/org/apache/carbondata/core/metadata/AbsoluteTableIdentifier.java

```diff
@@ -99,8 +98,9 @@ public static AbsoluteTableIdentifier fromTablePath(String tablePath) {
   }
 
   public String getTablePath() {
-    return getStorePath() + File.separator + getCarbonTableIdentifier().getDatabaseName() +
-        File.separator + getCarbonTableIdentifier().getTableName();
+    return getStorePath() + CarbonCommonConstants.FILE_SEPARATOR + getCarbonTableIdentifier()
```

Comment:
why need to change this? In another modification in this PR, you still use `File.separator` in `renameBadRecord`.